vars/mtaBuild.groovy

```diff
@@ -3,10 +3,11 @@ import static com.sap.piper.Prerequisites.checkScript
 import com.sap.piper.DownloadCacheUtils
 
 @Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/mtaBuild.yaml'
```

Comment:
Out of curiosity: what was wrong with the old coding? Do we have some coding conventions like exposing the metadata file as field?